tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -1486,41 +1489,39 @@ namespace tuplex {
         return wq.popCompletedTasks();
     }
 
-    // ensure output folder exists. => separate function here, b.c. it slows down pipeline by a lot!
-    void ensureOutputFolderExists(const URI& baseURI) {
-        std::string base; // base which to use to form string
-        std::string ext; // base
+    void LocalBackend::validateOutputSpecification(const URI& baseURI) {
+        std::string base;
```

Comment:
I think on the lambda-exp branch I wrote some function to normalize paths containing multiple /, should be probably called before doing the search with / and so on so some weird paths like file://////test////hello are caught.